### ISSUE TITLE
tableVersion in 'vhea' is confusing

### ISSUE THREAD

**Issue by mashabow** [fonttools/fonttools#540]
I got a dump of 'vhea' table from `SourceHanSans-Regular.otf` like this:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ttFont sfntVersion="OTTO" ttLibVersion="3.0">

  <vhea>
    <tableVersion value="1.0625"/>
...
```
It says the version is 1.0625, while this font has a 'vhea' table with version 1.1 (`0x00011000`). 

"Version Numbers" section in [the OpenType spec](https://www.microsoft.com/typography/otspec/otff.htm "The OpenType Font File") states as below.

> When a Fixed number is used as a version, the upper 16 bits comprise a major version number, and the lower 16 bits a minor. Tables with non-zero minor version numbers always specify the literal value of the version number since the normal representation of Fixed numbers is not necessarily followed. For example, the version number of 'maxp' table version 0.5 is 0x00005000, and that of 'vhea' table version 1.1 is 0x00011000.

**Comment by behdad** [comment]
Ouch!

Note that OpenType is not consistent about this. For example, GDEF table version 1.2 uses 0x00010002:

https://www.microsoft.com/typography/otspec/gdef.htm

Maybe we should remove the float conversion and go back to pure hex.

**Comment by behdad** [comment]
'head' table is also affected; though that one has only one version.

**Comment by behdad** [comment]
In the opentype-gx branch I changed Version of GSUB/GPOS/... to always be hex.  The ones reported in this issue still remain broken.
https://github.com/behdad/fonttools/commit/56c0a784d2da52c2ec125c0e1392e51d24b38f5c

**Comment by anthrotype** [comment]
Fixed in #711
